### PR TITLE
Updated to support VDK0.5.0

### PR DESCRIPTION
**Pull request by OsiJr**

### CODE REVIEW

File: src/vcBPA.cpp [hunk 1]
```diff
@@ -983,10 +987,16 @@ void vcBPA_CompareExport(vdkContext *pContext, vdkPointCloud *pOldModel, vdkPoin
...
+  vdkAttributeSet_Generate(&item.attributes, vdkSAC_ARGB, 1);
+  item.attributes.pDescriptors[item.attributes.count].blendMode = vdkABM_SingleValue;
+  item.attributes.pDescriptors[item.attributes.count].typeInfo = vdkAttributeTypeInfo_float32;
+  udStrcpy(item.attributes.pDescriptors[item.attributes.count].name, "udDisplacement");
+  ++item.attributes.count;
+
   item.sourceResolution = header.convertedResolution;
   item.pointCount = metadata.Get("SourcePointCount").AsInt64();
-  item.pointCountIsEstimate = true;
+  item.pointCountIsEstimate = false;
```
Comment (samsinsane) [inline src/vcBPA.cpp]: Probably could have deleted this instead.

File: src/vcRender.cpp [hunk 2]
```diff
@@ -1088,27 +1089,19 @@ udResult vcRender_RenderUD(vcState *pProgramState, vcRenderContext *pRenderConte
...
-    vdkRenderContext_ShowIntensity(pProgramState->pVDKContext, pRenderContext->udRenderContext.pRenderer, pProgramState->settings.visualization.minIntensity, pProgramState->settings.visualization.maxIntensity);
-    break;
-  case vcVM_Classification:
-    vdkRenderContext_ShowClassification(pProgramState->pVDKContext, pRenderContext->udRenderContext.pRenderer, (int *)pProgramState->settings.visualization.customClassificationColors, (int)udLengthOf(pProgramState->settings.visualization.customClassificationColors));
-    break;
-  default: //Includes vcVM_Colour
-    vdkRenderContext_ShowColor(pProgramState->pVDKContext, pRenderContext->udRenderContext.pRenderer);
-    break;
+    pModels = udAllocType(vdkRenderInstance, renderData.models.length, udAF_None);
+    pVoxelShaderData = udAllocType(vcUDRSData, renderData.models.length, udAF_None);
```
Comment (samsinsane) [inline src/vcRender.cpp]: Might be a good idea to validate these allocations.

File: azure-pipelines.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@ trigger:
     - '*'  # must quote since "*" is a YAML reserved character; we want a string
 
 variables:
-  linkedVDK: Euclideon_vdk0.4.1
+  linkedVDK: Pipeline_201911053
```
[the remark this answers is not in the document]
Comment (OsiJr) [reply azure-pipelines.yml]: VDK hasn't been changed yet

[the remark this answers is not in the document]
Comment (samsinsane) [reply azure-pipelines.yml]: Right oops, I seem to keep forgetting which repo we made build changes to.

**Pull request merged**